src/core/game.ts

```diff
@@ -125,13 +144,17 @@ export class Game {
     if (state.entities.players) {
       count = Object.keys(state.entities.players).length;
     }
-    player.pos.x = state.world.width / 4;
     if ( (count + 1) % 2 === 0) {
-      player.pos.x = player.pos.x + state.world.width / 2;
+      player.pos.x = 80;  
```

Comment:
I think it'd be better to actually have the positions described in the `Map` definition in `wall.ts`.   The validity of the player start positions depends on the map, so it should be specified by the map. 

```js
// State for the backup data for a wall
export interface StoredWallState  {
  pos: {
    x: number;
    y: number;
  };
  width: number;
  height: number;
  sprite: WallSprite;
}

export interface StoredMapState {
  walls: StoredWallState[];
  startPositions: {x: number; y: number}[];   // <- don't know if that syntax works, but you get the idea
}

// List of maps, stores the minimal amount of data needed so that
// this could be moved into a JSON file to backup maps
export const MapCatalog: StoredMapState[] = [{
  walls: [
    {pos: {x: 380, y: 260}, width: 200, height: 200, sprite: WallSprite.BUNKER_2x2_1},
    {pos: {x: 170, y: 145}, width: 140, height: 70, sprite: WallSprite.BUNKER_2x1_1},
    {pos: {x: 205, y: 470}, width: 70, height: 140, sprite: WallSprite.BUNKER_1x2_1},
    {pos: {x: 650, y: 505}, width: 140, height: 70, sprite: WallSprite.BUNKER_2x1_2},
    {pos: {x: 685, y: 110}, width: 70, height: 140, sprite: WallSprite.BUNKER_1x2_1},
  ],
  startPositions: [
    {x: ?, y: ?},
    {x: ?, y: ?},
    {x: ?, y: ?},
    {x: ?, y: ?},
  }
}];
```

`game.ts` would have to be changed appropriately